larq/layers_base.py

```diff
@@ -34,35 +64,25 @@ def __init__(
                 "may result in starved weights (where the gradient is always zero)."
             )
 
+    def get_quantizer(self, name):
+        if name == "kernel":
+            return self.kernel_quantizer
+        return None
```

Comment:
Me too

Reply:
@lgeiger I probably shouldn't have put this as suggested change because the 

```python
        if name == "kernel":
            return self.kernel_quantizer
```

is still there now.